### PR TITLE
Adding reviews

### PR DESCRIPTION
**Pull request by sbeleidy**

### CODE REVIEW

File: pages/content/[contentId].js [hunk 1]
```diff
@@ -41,13 +42,48 @@ export default function ContentPage() {
...
 
   const db = getFirestore(firebaseApp);
 
   const router = useRouter();
   const { contentId } = router.query;
 
+  const updateReview = async (e) => {
+    e.preventDefault();
+    if (contentPurchased && Object.keys(userInDb).length > 0) {
+      console.log("User and purchased content");
```
Comment (ynadeem) [inline pages/content/[contentId].js:60]: Remove the console.log

File: pages/content/[contentId].js [hunk 2]
```diff
@@ -41,13 +42,48 @@ export default function ContentPage() {
...
+          }
+        );
+      } else {
+        const newReview = {
+          content: doc(db, "content", contentId),
+          user: doc(db, "users", userInDb.id),
+          rating: newUserReviewRating,
+          description: reviewDescriptionInput.current.value,
+        };
+        console.log(newReview);
```
Comment (ynadeem) [inline pages/content/[contentId].js:76]: Remove console.log

File: pages/content/[contentId].js [hunk 3]
```diff
@@ -484,15 +588,12 @@ export default function ContentPage() {
...
-                    <svg
-                      xmlns="http://www.w3.org/2000/svg"
-                      className="h-4 w-4 inline-block"
-                      viewBox="0 0 20 20"
-                      fill="currentColor"
-                    >
-                      <path d="M9.049 2.927c.3-.921 1.603-.921 1.902 0l1.07 3.292a1 1 0 00.95.69h3.462c.969 0 1.371 1.24.588 1.81l-2.8 2.034a1 1 0 00-.364 1.118l1.07 3.292c.3.921-.755 1.688-1.54 1.118l-2.8-2.034a1 1 0 00-1.175 0l-2.8 2.034c-.784.57-1.838-.197-1.539-1.118l1.07-3.292a1 1 0 00-.364-1.118L2.98 8.72c-.783-.57-.38-1.81.588-1.81h3.461a1 1 0 00.951-.69l1.07-3.292z" />
-                    </svg>
+                    <Rating
+                      value={review.data.rating}
```
Comment (ynadeem) [inline pages/content/[contentId].js:592]: Can this be rounded to 1 or 2 decimal places? We don't want to render a rating of 3.666666666666667 stars to the user. Did some quick searching on Google and this screenshot seems to be the accepted solution:

image

**Review by ynadeem — CHANGES_REQUESTED** [review]
Please review my comments

File: pages/content/[contentId].js (hunk 3 above)
Comment (sbeleidy) [reply pages/content/[contentId].js:592]: Lol why not send me a link? This is super copy pastable 😛 
I'll do some variation of this. I'll also apply it to everywhere we use ratings

**Review by ynadeem — APPROVED** [review]
Looks great!

**Pull request merged**